lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -445,7 +445,8 @@ protected function transformSignup($data, $current) {
    */
   protected function transformUser($data) {
     return [
-      'id' => $data->id,
+      'id' => $data->drupal_id,
```

Comment:
So this might actually cause a bunch of problems. A while back, we decided that in the API response `id` would be the NS id and we would not expose the Drupal ID since we're moving away from from Drupal IDs as NS takes over. So I think this might mess a few things up (particularly with the `/reportback` and `/reportback-items` endpoints when passing the [`load_user=true`](https://github.com/DoSomething/phoenix/wiki/API#reportbacks) param that returns the NS user and populates the prior `$data->id`... I'm worried this might break things already in play (ie: mobile app)